src/util/index.js

```diff
@@ -25,6 +25,15 @@ var util = {
     /** {@link Version} */
     Version: require("./Version"),
 
+    /** expose some dependencies for hooks */
```

Comment:
Those are standard node modules. You should not need to export them with our internal mechanism. Otherwise everything LGTM!